src/main/java/com/minecolonies/compatibility/Compatibility.java

```diff
@@ -25,10 +30,14 @@ private Compatibility()
      */
     public static boolean getMiningLevelCompatibility(@Nullable final ItemStack stack, @Nullable final String tool)
     {
-        if (ToolBrokenCheck.checkTinkersBroken(stack))
-        {
-            return false;
-        }
-        return true;
+        return !ToolBrokenCheck.checkTinkersBroken(stack);
+    }
+    public static boolean getTreeCompatibility(@NotNull final World world, @NotNull final BlockPos pos)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)